projects/storefrontlib/shared/components/captcha/config/default-captcha-api-config.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CaptchaApiConfig } from './captcha-api-config';
+import { Subject } from 'rxjs';
+
+export const defaultCaptchaApiConfig: CaptchaApiConfig = {
```

Comment:
Lets try and move this to a service like `GoogleV2CaptchaService` that extends our `CaptchaService`. 

You can find examples of how this can be done in our `TagManagementService` or `TMS` (see `feature-libs/tracking`) which does something similar with third-party providers allowing users to import `Google Tag Manager` or  `Adobe Experience Platform`